### PR TITLE
maenad submission

### PR DESCRIPTION
**Pull request by maenad**
### New Stylesheet Submission:

- [x] Have you followed the guidelines in our Contributing document?
- [x] Have you checked to ensure there aren't other files using the same name as yours?
- [x] Have you filled in at least the required data for `title`, `author`, and `stylesheet`?
- [x] Is the value of `stylesheet` a full URL that is publicly accessible and renders an unminified, compiled CSS file?
- [x] Do your styles pass proper contrast for accessibility?
- [x] Have you included `@media (prefers-reduced-motion: reduce)` to remove animations on `*,*::before,*::after`?
- [x] Did you style the `.skip-link`?
- [x] Have you checked on a real mobile device to ensure your responsive design doesn't cause overlaps or overflow scrolling?

### CODE REVIEW

File: src/_data/styles/maenad.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    // Required
```
Comment (5t3ph) [inline src/_data/styles/maenad.json:2]:
```suggestion
    // Required
```

File: src/_data/styles/maenad.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+    "title": "Dot grid",
+    "author": "Abigail Colina",
+    "stylesheet": "https://raw.githubusercontent.com/maenad/stylestage-submission/master/style.css",
+    // Optional
```
Comment (5t3ph) [inline src/_data/styles/maenad.json:6]:
```suggestion
    // Optional
```

**Review by 5t3ph — CHANGES_REQUESTED** [review]

**Comment by 5t3ph** [comment]
Hi @maenad - a few things for you to check!

First, I wanted to make sure you were ok with the content getting this narrow before breaking to a wider width?
image

The color for links and buttons needs to be darker on `:hover`:
image
image

Related, ideally when used as the button background, the background needs 3:1 contrast or higher with it's surroundings (or have a border that meets that condition), while keeping the button text with 4.5:1 contrast with the button background:
image
image

I have 20/20 eyesight, but even for me this was hard to read - would you mind keeping it at a weight of `400`?
image

Lastly, the `:focus` style for especially text links is not very visible especially since something is making it get cutoff a bit - can you provide a custom style? It needs to meet at least 3:1 contrast. Once you resolve the `:hover` you _could_ include `:focus` for that rule as well:
image

**Comment by maenad** [comment]
Hello! Thanks for the quick answer.

Not sure why it is showing this way, in my file looks like this on desktop and mobile, not sure what could be wrong.

image
image

For the rest, I am on my way to fix them, thanks for the input!

**Comment by 5t3ph** [comment]
@maenad for the width I checked it by manually resizing the browser so it gets to be that narrow width on an in-between breakpoint if that helps!

**Comment by maenad** [comment]
I fixed them on my main repo and the changes should appear on the same link now. Please check and let me know if it got fixed.

Thanks a lot for the prompt response, have a great weekend!

**Comment by maenad** [comment]
> 
> 
> I fixed them on my main repo and the changes should appear on the same link now. Please check and let me know if it got fixed.
> 
> Thanks a lot for the prompt response, have a great weekend!

Not sure if I have to do anything else or create a new PR, please let me know to make amends if neded.

**Comment by 5t3ph** [comment]
Hi @maenad - I redeployed the preview but a few changes were missed - the button background contrast still needs adjusted (3:1 against white basically) and improving `:focus` style for in-text (like in-paragraph) links. For the `:focus`, you could update your `:hover` style to include the same styling on `:focus`. Thanks!

I also just noticed that the list styling was completely removed from the "Steps to Contribute" which removes some clarity from reading the content. It's ok to remove the bullets but perhaps at least add back in some space so it doesn't appear as one paragraph.
image

**Comment by maenad** [comment]
Changes have been made. Just in case there is any issue with the link, this should be it -- https://raw.githubusercontent.com/maenad/stylestage-submission/master/style.css

For reference, these got fixed:

image
There you can see the Steps to Contribute look better and the hovered links have more contrast.
I used this website to corroborate the contrast between elements - https://webaim.org/resources/contrastchecker/

image
Buttons hover also got fixed to avoid contrast issues and to match the home one.

**Comment by 5t3ph** [comment]
@maenad thanks for sticking with this! Here's the contrast I'm still concerned about:
image
image

One way to resolve is to add a border that meets the contrast instead of changing the whole button background but as a UI element these "buttons" need to have at least 3:1 contrast with their surroundings 👍

**Comment by maenad** [comment]
Hello again! Sorry, my bad, I had not understood the request. I fixed it now, feel free to check it whenever you have time.
Thanks for taking the time to be so thorough with this, it's great!
-- 
Everything should look like this

image

Have a wonderful weekend.

**Review by 5t3ph — APPROVED** [review]
@maenad good to go, great work!

Thank you for participating! If you haven't already, consider subscribing for updates to learn about new styles and features: https://stylestage.dev/subscribe/

**Comment by maenad** [comment]
Thanks a lot for the opportunity!

**Pull request merged**